README.md

```diff
@@ -54,11 +54,15 @@
 You will first need to start two databases, postgres and mongo. Postgres will be used by the server to store general data and Mongo will store the data of the applications that you run.
 You can do this by using the 
 
-`docker compose up` 
+`docker compose up -d` 
```

Comment:
The new lines are not needed since it's not a list anymore

Reply:
It wasn't a list before eather :p 
Fixed

Reply:
My bad, it was the `-` of the diff ^^